apitools/org.eclipse.pde.api.tools/src/org/eclipse/pde/api/tools/internal/model/ApiElement.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2008, 2009 IBM Corporation and others.
+ * Copyright (c) 2008, 2023 IBM Corporation and others.
```

Comment:
FYI - updating this year is optional; I personally find it a waste of effort. If you like to do this, that is fine but in case you also find it non-value add, you can skip this step.